Anlab.Jobs.SendMail/Program.cs

```diff
@@ -55,7 +55,7 @@ static void Main(string[] args)
 
 
             // Get all messages that we haven't tried to send yet
-            var messagesToSend = dbContext.MailMessages.Where(x => x.Sent == null).ToList();
+            var messagesToSend = dbContext.MailMessages.Where(x => x.Sent == null || x.Sent == false).ToList();
```

Comment:
@jSylvestre go ahead and update the comment too since now we are getting unsent and failed messages